app/scripts/controllers/pod.js

```diff
@@ -87,6 +87,49 @@ angular.module('openshiftConsole')
       }
     };
 
+    /**
+     *  Will set the newTerm's isActive/isUsed to true, while deactivating the old's
+     */
+    $scope.onTerminalSelectChange = function(newTerm) {
+      // Deactivate all terminals
```

Comment:
We do but the connection could die from the server side or somewhere in
between, and that terminal may not be visible when it happens.

On Jul 11, 2016 5:14 PM, "Sam Padgett" <notifications@github.com> wrote:

> In app/scripts/controllers/pod.js
> <https://github.com/openshift/origin-web-console/pull/66#discussion_r70338365>
> :
>
> > @@ -87,6 +87,49 @@ angular.module('openshiftConsole')
> >        }
> >      };
> >
> > +    /**
> > +     *  Will set the newTerm's isActive/isUsed to true, while deactivating the old's
> > +     */
> > +    $scope.onTerminalSelectChange = function(newTerm) {
> > +      // Deactivate all terminals
>
> I'm not sure what isActive means. Isn't this just isVisible? We still
> keep the terminal session alive, right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/66/files/931c6b14e057a051f4eb1826e8c25ebd44f2f3be#r70338365>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABZk7Rt0Hxlk5kBDlQxiLhm2UTGsSjvXks5qUrIhgaJpZM4I2oUA>
> .
>